packages/generators/src/init-generator.ts

```diff
@@ -86,4 +107,24 @@ export default class InitGenerator extends CustomGenerator {
         this.utils.logger.log(`${blue('ℹ INFO ')} Initialising project...`);
         handlers[this.template].generate(this);
     }
+
+    public end(): void {
+        // Prettify configuration file if possible
+        try {
+            // eslint-disable-next-line node/no-extraneous-require, @typescript-eslint/no-var-requires
+            const prettier = require('prettier');
+            const source = readFileSync(this.configurationPath, { encoding: 'utf8' });
+            const formattedSource = prettier.format(source, {
+                singleQuote: true,
+                trailingComma: 'all',
+                printWidth: 140,
+                tabWidth: 4,
+                parser: 'babel',
+            });
```

Comment:
I think better do it without our options, because some developers can prefer other options, prettier should read configuration(s) by default

Reply:
Okay, on it